sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Cluster Lifecycle Charter
+
+## Scope
+
+SIG Cluster Lifecycle’s objective is to improve the deployment, configuration management, and upgrade story for Kubernetes clusters and relevant infrastructure.  (Note best practices)
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Improving the Kubernetes UX for cluster administration
+- Kubernetes cluster lifecycle related features and issues
+- Tools that assist in bootstrapping, mutating, and upgrading Kubernetes control plane components. 
+- Kubernetes tools used to manage infrastructure related to provisioning and upgrading nodes
+- Tools that assist in management of configuration of Kubernetes components
+
+#### Code, Binaries and Services
+
+- Everything that falls in the scope of the SIG.
+- Tools that are provider specific implementation for infrastructure management.
+- Core add-ons that are required for cluster bootstrapping.
+
+#### Cross-cutting and Externally Facing Processes
+
+- This SIG works closely with sig-release during the end of the cycle, because of how the release is currently managed. The process requires a lock-step coordination which is outlined [here](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md)
+- The SIG endorses third party cluster add-ons for networking, network policy, service discovery, etc. The SIG maintains the health check of container images for some add-ons that are required for cluster bootstrapping. While the SIG could provide support to users that have add-on related issues, the SIG can decide to delegate issues to the add-on maintainers or other SIGs.
```

Comment:
endorses is strong.  How about "recommends and verifies compatibility of critical ..."